configuration/concepts/commons.csv

```diff
@@ -126,11 +128,13 @@ a82e292b-7bd3-4bf1-b6d8-d89c8b8cabff,,,Two,Deux,Two,Deux,,,Misc,N/A,,,,,,,,,,,
 877b6c50-7960-4be6-862f-c9ffd2f65afb,,,Three,Trois,Three,Trois,,,Misc,N/A,,,,,,,,,,,
 d29d0005-5e23-48cd-86ac-1ff9cd36dd13,,,Four,Quatre,Four,Quatre,,,Misc,N/A,,,,,,,,,,,
 3098581b-4ce3-4064-8e87-facb1d7a5ff5,,,Five plus,Cinq plus,Five plus,Cinq plus,,,Misc,N/A,,,,,,,,,,,
-5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,Zero,Zéro,0,0,,,Misc,N/A,,,,,,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,Zero,Zero,0,0,,,Misc,N/A,,,,,,,,,,,
 e7c0fefc-96e8-4ccb-b24b-92ec2024f579,,CIEL:1362,One plus,Un plus,+,+,,,Misc,N/A,,,,,,,,,,,
 9f14f988-be10-419b-b947-0508664b7ef2,,CIEL:1363,Two plus,Deux plus,++,++,,,Misc,N/A,,,,,,,,,,,
 a979848e-4b09-4319-bdd5-ce7d38e0b686,,CIEL:1364,Three plus,Trois plus,+++,+++,,,Misc,N/A,,,,,,,,,,,
 30c7d366-2192-45bf-a0df-112258bc0c1e,,CIEL:1365,Four plus,Quatre plus,++++,++++,,,Misc,N/A,,,,,,,,,,,
-c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Number of Visit,Nombre de visites,Number of Visit,Nombre de visites,,,Misc,Coded,,One;Two;Three;Four;Five plus,,,,,,,,,
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Number of Visit,Nombre de visites,Number of Visit,Nombre de visites,,,Misc,Coded,,Une;Deux;Trois;Quatre;Cinq plus,,,,,,,,,
```

Comment:
I think this should be renamed:
'Visit number" / "Numéro de la visite"

Reply:
Done